pro/src/screens/OfferType/OfferTypeButton/OfferTypeButton.module.scss

```diff
@@ -0,0 +1,39 @@
+.button {
+    background-color: transparent;
+    border: 1px solid $black;
+    border-radius: 8px;
+    padding: rem(32px) rem(16px) rem(16px);
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+    width: rem(231px);
+    font-size: rem(15px);
+
+    &-checked {
+        position: absolute;
+        top: rem(16px);
+        left: rem(16px);
+    }
```

Comment:
du coup la c'est l'état `is-checked` ? ou l'élément `check-mark`, j'utiliserait plutôt `&-check-mark { ... }` du coup pour rendre l'intention plus claire